README.md

```diff
@@ -269,13 +269,29 @@ The following commands can be issued as batch commands (refer to [How to send ba
     ```
 - Send an energy offer with price in cents:
     ```python
-    offer_energy(area_uuid, energy, price_cents, replace_existing)
+    offer_energy(area_uuid, energy, price_cents, replace_existing, attributes, requirements)
     ```
 - Send an energy offer with energy rate in cents/kWh:
     ```python
-    offer_energy_rate(area_uuid, energy, rate_cents_per_kWh, replace_existing)
+    offer_energy_rate(area_uuid, energy, rate_cents_per_kWh, replace_existing, attributes, requirements)
     ```
 ---
+### Available attributes/requirements
+The offer/bid functions accept attributes (dict attributes of the offer or bid) | requirements (list of requirements dicts that at least 1 of them need to be satisfied in the matching process)
+Currently supported attributes:
+- Offers:
+    - `energy_type`: Energy type of the offer
+- Bids: No supported bids' attributes at the moment
+
+Currently supported requirements:
+- Offers:
+    - `trading_partners`: preferable area ids to match with.
+- Bids:
+    - `energy_type`: Energy types that the bid prefers to consume.
+    - `trading_partners`: preferable area ids to match with.
+    - `energy`: Energy that the bid prefers to consume.
+    - `price`: Trade rate that the bid prefers to accept.
+    
```

Comment:
Nit: I tried to rephrase this a bit:
```suggestion
### Attributes and requirements

A finer control over the issued bids and offers can be achieved through the `attributes` and `requirements` parameters of the bid and offer methods.

#### `attributes`

A dictionary of attributes that describe the offer or bid. Currently supported attributes:

- Offers:
    - `energy_type`: energy type of the offer

- Bids: Not supported at the moment

#### `requirements`

A list of dictionaries containing requirements for the offer or bid. At least one of the specified requirements needs to be satisfied in the matching process. Currently supported requirements:

- Offers:
    - `trading_partners`: IDs of the areas with which the offer has to be matched

- Bids:
    - `energy_type`: energy types that the bid prefers to consume
    - `trading_partners`: IDs of the areas with which the bid prefers to be matched
    - `energy`: energy that the bid prefers to consume
    - `price`: trade rate that the bid prefers to accept
```
Feel free to use it or discard it if you don't like it!